### PR TITLE
NEW: add .travis.yml

### PR DESCRIPTION
**Pull request by bugant**
TODO:

- [x] fix or comment disable failing tests

### CODE REVIEW

**Comment by bugant** [comment]
@potomak do you think there is a way to make that tests that some time fails, more reliable?

**Comment by bugant** [comment]
@potomak I think there was a wrong test. Please check commit https://github.com/issuehunter/issuehunter/pull/21/commits/c40821c7f527d6176b6c36e43591fd305cc64bc1 and tell me if I'm wrong.

File: test/issuehunter.js [hunk 1]
```diff
@@ -1284,7 +1284,7 @@ contract('Issuehunter', function (accounts) {
         }).then(function () {
           return verifyPatch(issueId, author, ref, patchVerifier)
         }).then(function () {
-          return increaseTime(60 * 60 * 24 * 8)
+          return increaseTime((60 * 60 * 24 * 7) - 1)
```
Comment (potomak) [inline test/issuehunter.js]: This should be `((60 * 60 * 24 * 8) - x)`, where `x` is a small number of seconds, ideally between 2 and 5.

You should do the same in the `right before the pre-reward period end` context, and mark https://github.com/issuehunter/issuehunter/issues/15 as complete.

**Comment by potomak** [comment]
> do you think there is a way to make that tests that some time fails, more reliable?

@bugant yes, but the only way I can think of will make the test less strict.

File: test/issuehunter.js (hunk 1 above)
Comment (bugant) [reply test/issuehunter.js]: @potomak isn't it 7 by default?

Comment (potomak) [reply test/issuehunter.js]: It's 1 day (pre-reward period) + 7 days (reward period).

**Review by potomak — APPROVED** [review]
🥇

**Comment by potomak** [comment]
@bugant why did you close this PR instead of merging it?

**Pull request merged**